tests/conftest.py

```
@@ -84,6 +89,39 @@ def temp_dir():
+
+
+@pytest.fixture
+def make_relays(temp_dir):
+    """Factory that builds a Relays instance writing into a per-test temp directory.
+
+    Shared setup for the directory-authority integration tests (and any other
+    test that needs a real Relays object without hitting the network).
+    """
+    from allium.lib.relays import Relays
+
+    def _make(relay_data, onionoo_url=TEST_DETAILS_URL):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_make`

(ANN202)